components/listingsPage/redux/reducer.js

```diff
@@ -0,0 +1,44 @@
+import {
+  GET_FIRMS_SUCCESS,
+  GET_OFFERINGS_SUCCESS,
+  FILTER_OFFERING,
+} from "./constants";
+
+let initialState = {
+  firms: [],
+  offerings: [],
+};
+
+const ListingsReducer = (state = initialState, action) => {
+  switch (action.type) {
+    case "GET_FIRMS_STARTED":
```

Comment:
It looks to me like  this particular action is never dispatched. If so it could be removed. Otherwise it's no big deal.

Reply:
Ouch! My bad, its actually one of the cleanups i should have done. Removed.